src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -157,13 +159,36 @@ const ViroSoundAnd3DObject = (item) => {
             onLoadStart={() => setIsObjectLoading(true)}
             onLoadEnd={() => setIsObjectLoading(false)}
             onError={() => alert(texts.augmentedReality.arShowScreen.objectLoadErrorAlert)}
+            shadowCastingBitMask={2}
             animation={{
               loop: true,
               name: object?.animationName,
               run: isStartAnimationAndSound
             }}
           />
         ))}
+
+      <ViroSpotLight
+        castsShadow
+        direction={[0, -1, -0.2]}
+        influenceBitMask={2}
+        innerAngle={5}
+        outerAngle={45}
+        position={[0, 3, 1]}
+        shadowFarZ={5}
+        shadowMapSize={2048}
+        shadowNearZ={2}
+        shadowOpacity={0.6}
+      />
+
+      <ViroQuad
+        arShadowReceiver
+        height={1000}
+        width={1000}
+        lightReceivingBitMask={2}
+        position={[0, 0, 0]}
+        rotation={[-90, 0, 0]}
```

Comment:
which props do we want editable from the CMS? `height`, `width`, `position`?

Reply:
Only the `position` feature should be adjustable. 👍

Reply:
Do we plan to add a setting for the degree of `SpotLight` as we discussed yesterday?

Reply:
we wanted to ask the artists first on Monday, right?

Reply:
new fields are done in the cms